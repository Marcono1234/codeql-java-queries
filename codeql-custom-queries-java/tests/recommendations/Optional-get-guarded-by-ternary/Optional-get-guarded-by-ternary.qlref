./recommendations/Optional-get-guarded-by-ternary.ql
