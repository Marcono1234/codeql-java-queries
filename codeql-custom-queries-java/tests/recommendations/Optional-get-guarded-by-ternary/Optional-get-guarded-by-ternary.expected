| Test.java:9:13:9:48 | ...?...:... | Could use Optional.orElseGet(...) instead |
| Test.java:10:13:10:56 | ...?...:... | Could use Optional.orElseGet(...) instead |
| Test.java:11:13:11:46 | ...?...:... | Could use Optional.orElseGet(...) instead |
| Test.java:12:13:12:54 | ...?...:... | Could use Optional.orElseGet(...) instead |
| Test.java:17:13:17:52 | ...?...:... | Could use Optional.orElseGet(...) instead |
| Test.java:18:13:18:52 | ...?...:... | Could use Optional.orElseGet(...) instead |
| Test.java:19:13:19:50 | ...?...:... | Could use Optional.orElseGet(...) instead |
| Test.java:20:13:20:50 | ...?...:... | Could use Optional.orElseGet(...) instead |
| Test.java:25:13:25:48 | ...?...:... | Could use Optional.orElseGet(...) instead |
| Test.java:26:13:26:51 | ...?...:... | Could use Optional.orElseGet(...) instead |
| Test.java:27:13:27:46 | ...?...:... | Could use Optional.orElseGet(...) instead |
| Test.java:28:13:28:49 | ...?...:... | Could use Optional.orElseGet(...) instead |
| Test.java:33:13:33:50 | ...?...:... | Could use Optional.orElseGet(...) instead |
| Test.java:34:13:34:52 | ...?...:... | Could use Optional.orElseGet(...) instead |
| Test.java:35:13:35:48 | ...?...:... | Could use Optional.orElseGet(...) instead |
| Test.java:36:13:36:50 | ...?...:... | Could use Optional.orElseGet(...) instead |
